be/src/olap/storage_engine.cpp

```diff
@@ -357,6 +358,15 @@ Status StorageEngine::get_all_data_dir_info(std::vector<DataDirInfo>* data_dir_i
     return res;
 }
 
+Status StorageEngine::get_log_dir_info(LogDirInfo* log_dir_infos) {
+    Status res = Status::OK();
+    _log_dir->health_check();
+    _log_dir->update_capacity();
+    LogDirInfo dir_info = _log_dir->get_dir_info();
```

Comment:
Why not just pass the log_dir_infos pointer as an argument to get_dir_info()?

Reply:
done